selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/projectexplorer/RenameProjectTest.java

```diff
@@ -10,31 +10,35 @@
  */
 package org.eclipse.che.selenium.projectexplorer;
 
+import static org.eclipse.che.commons.lang.NameGenerator.generate;
+import static org.eclipse.che.selenium.core.constant.TestProjectExplorerContextMenuConstants.RENAME;
+import static org.eclipse.che.selenium.pageobject.ProjectExplorer.FolderTypes.PROJECT_FOLDER;
+
 import com.google.inject.Inject;
 import java.net.URL;
 import java.nio.file.Paths;
 import org.eclipse.che.selenium.core.client.TestProjectServiceClient;
-import org.eclipse.che.selenium.core.constant.TestProjectExplorerContextMenuConstants;
+import org.eclipse.che.selenium.core.constant.TestMenuCommandsConstants.Edit;
 import org.eclipse.che.selenium.core.project.ProjectTemplates;
 import org.eclipse.che.selenium.core.workspace.TestWorkspace;
 import org.eclipse.che.selenium.pageobject.AskForValueDialog;
 import org.eclipse.che.selenium.pageobject.Ide;
-import org.eclipse.che.selenium.pageobject.Loader;
+import org.eclipse.che.selenium.pageobject.Menu;
 import org.eclipse.che.selenium.pageobject.ProjectExplorer;
 import org.testng.annotations.BeforeClass;
 import org.testng.annotations.Test;
 
 /** @author Andrey Chizhikov */
 public class RenameProjectTest {
-  private static final String PROJECT_NAME = RenameProjectTest.class.getSimpleName();
-  private static final String NEW_PROJECT_NAME = "new-name-project";
```

Comment:
Let's use "new-" prefix here in new project name to have more information on screenshot in case of failure.